src/polycap-source.c

```diff
@@ -436,7 +436,7 @@ polycap_source* polycap_source_new_from_file(const char *filename, polycap_error
 // for a given array of energies, and a full polycap_description, get the transmission efficiencies.
 polycap_transmission_efficiencies* polycap_source_get_transmission_efficiencies(polycap_source *source, int max_threads, int n_photons, bool leak_calc, polycap_progress_monitor *progress_monitor, polycap_error **error)
 {
-	int i, j;
+	int i, j=0;
 	int64_t sum_iexit=0, sum_irefl=0, sum_not_entered=0, sum_not_transmitted=0;
```

Comment:
Als ik u was, zou ik gewoon `int j=0` brengen naar line 731, net voor de omp for loop begint, `j` wordt toch niet eerder gebruikt.

Reply:
Mja, maar kdacht datie dan misschien lastig ging doen over lijn 686, waar ik die j als private verklaar voor elke thread. Dus kzou het precies daar nog voor plaatsen, en dan zet ik het liever bij al mijn andere variabelen?

Reply:
Verwijder dan de referentie naar `j` in private: als de variabele gedefinieerd is binnen een omp block is ze hoe dan ook private.